doc/manuals/user/ngsiv2_implementation_notes.md

```diff
@@ -275,7 +275,9 @@ From lowest to highest:
 The NGSIv2 specification describes in section "Subscriptions" the rules that trigger notifications
 corresponding to a given subscription, based on updates to the entities covered by the subscription.
 Apart from that kind of regular notifications, Orion may send also an initial notification at
-subscription creation/update time. Check details in the document about [initial notifications](initial_notification.md)
+subscription creation/update time. 
+Initial Notification can be configurable using a new URI parameter option  `skipInitialNotification`. For instance `POST /v2/subscriptions?options=skipInitialNotification`. 
+Check details in the document about [initial notifications](initial_notification.md)
 
```

Comment:
I'd suggest to add 2 line breaks (i.e. text rendered in different paragraph inserting a blank line in the .md file) in the following point:

![imagen](https://user-images.githubusercontent.com/1534240/51265088-38416f00-19b8-11e9-9ed1-99021fe8dc23.png)

In addition, a small capitalization issue: "Initial Notification" -> "Initial notification" (N -> n)